[2024-12-11T21:25:11.150+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-11T21:25:11.184+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_and_store_google_books.fetch_book_data scheduled__2024-10-21T00:00:00+00:00 [queued]>
[2024-12-11T21:25:11.195+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_and_store_google_books.fetch_book_data scheduled__2024-10-21T00:00:00+00:00 [queued]>
[2024-12-11T21:25:11.197+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-12-11T21:25:11.253+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): fetch_book_data> on 2024-10-21 00:00:00+00:00
[2024-12-11T21:25:11.277+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1771) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-11T21:25:11.279+0000] {standard_task_runner.py:72} INFO - Started process 1797 to run task
[2024-12-11T21:25:11.284+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fetch_and_store_google_books', 'fetch_book_data', 'scheduled__2024-10-21T00:00:00+00:00', '--job-id', '1046', '--raw', '--subdir', 'DAGS_FOLDER/dag2.py', '--cfg-path', '/tmp/tmptekkwcfy']
[2024-12-11T21:25:11.286+0000] {standard_task_runner.py:105} INFO - Job 1046: Subtask fetch_book_data
[2024-12-11T21:25:11.402+0000] {task_command.py:467} INFO - Running <TaskInstance: fetch_and_store_google_books.fetch_book_data scheduled__2024-10-21T00:00:00+00:00 [running]> on host bb74d1700244
[2024-12-11T21:25:11.558+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fetch_and_store_google_books' AIRFLOW_CTX_TASK_ID='fetch_book_data' AIRFLOW_CTX_EXECUTION_DATE='2024-10-21T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-21T00:00:00+00:00'
[2024-12-11T21:25:11.563+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-12-11T21:25:11.564+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-12-11T21:25:11.564+0000] {logging_mixin.py:190} INFO - Current task name:fetch_book_data state:running start_date:2024-12-11 21:25:11.185062+00:00
[2024-12-11T21:25:11.565+0000] {logging_mixin.py:190} INFO - Dag name:fetch_and_store_google_books and current dag run status:running
[2024-12-11T21:25:11.565+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-11T21:25:13.509+0000] {logging_mixin.py:190} INFO - Failed to fetch data. Status code: 429
[2024-12-11T21:25:13.540+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-11T21:25:13.562+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-11T21:25:13.563+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=fetch_and_store_google_books, task_id=fetch_book_data, run_id=scheduled__2024-10-21T00:00:00+00:00, execution_date=20241021T000000, start_date=20241211T212511, end_date=20241211T212513
[2024-12-11T21:25:13.589+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-12-11T21:25:13.590+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-12-11T21:25:13.591+0000] {logging_mixin.py:190} INFO - Dag name:fetch_and_store_google_books queued_at:2024-12-11 21:25:05.191163+00:00
[2024-12-11T21:25:13.591+0000] {logging_mixin.py:190} INFO - Task hostname:bb74d1700244 operator:PythonOperator
[2024-12-11T21:25:13.628+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-11T21:25:13.686+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-11T21:25:13.689+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
